src/rust/ide/enso-protocol/src/common.rs

```diff
@@ -0,0 +1,4 @@
+//! Commons of JSON-RPC-based Enso services.
```

Comment:
The name `common` doesnt tell anything what it contains. We should not name files "common", "utils", nor "manager" if possible. I would never tell that a file "common" contain some date time utils.